.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Provide a code example and any sample input data (e.g. an H5AD) as an attachment to reproduce this behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```

Comment:
Could possibly combine this section with "Describe the bug"

Reply:
Done